numba/cuda/tests/cudapy/test_inspect.py

```diff
@@ -91,13 +92,17 @@ def foo(x, y):
         self.assertIn("foo", asmdict[float64, float64])
 
     def _test_inspect_sass(self, kernel, name, sass):
+        if not NVVM().is_nvvm70:
+            self.skipTest("lineinfo not generated for NVVM 3.4")
```

Comment:
Whilst I'd normally mention that this also ends up skipping the tests that specific SASS instructions appear later on in the test, which should work even for NVVM 3.4, it doesn't matter in this case because we're going to drop support for all NVVM 3.4-based toolkits before the next release.